sympy/matrices/common.py

```diff
@@ -1956,6 +1956,71 @@ def permute(self, perm, orientation='rows', direction='forward'):
         if orientation == 'cols':
             return self._eval_permute_cols(perm)
 
+#start
+
+ def rot90(self, k=1):
+        """Rotates Matrix by 90 degrees
```

Comment:
You should also add docs for the usage of `k`. Which value of `k` is used for which functionality?

Reply:
I didn't follow you. Could you please elaborate?